reboot_pending/reboot_pending.dsc.resource.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/bundled/resource/manifest.json",
+    "description": "Returns info about pending reboot.",
+    "type": "Microsoft/RebootPending",
```

Comment:
Should be `Microsoft.Windows` since it's Windows specific.

Reply:
Updated.